docs/en/operations/settings/settings.md

```diff
@@ -4203,6 +4203,62 @@ Default value: `0`.
 Use this setting only for backward compatibility if your use cases depend on old syntax.
 :::
 
+## session_timezone {#session_timezone}
```

Comment:
there is already a setting called `timezone` in `config.xml`. This may also introduce some mess. 
Also, don't remember exactly why, but I **_had_** to change the name, I used to call it `timezone`, but there were some errors. Maybe we can come up with a better naming option :) 
maybe something like `user_defined_timezone` or `user_timezone` would be fine

Reply:
Yes, setting `timezone` in "config.xml" *is* the global/server-level timezone.

As far as I see, there is no precedence for an equally named global/session setting, but I don't think anything can mess up. It would be very appreciated if you could make a try to rename `session_timezone` to `timezone` in a separate PR, after this one is merged.

(`user_defined_timezone` / `user_timezone` ar a bit too chatty for my taste)

Reply:
yes, agree

Reply:
It actually would be nicer to rename server's setting to `server_timezone`, but I guess it is a really bad option

Reply:
It is a bad idea :-) (and unfortunately settings can't be aliased as of now, so we can't deprecate setting "timezone").

Reply:
Ok, after this is merged I can try to change it to `timezone`